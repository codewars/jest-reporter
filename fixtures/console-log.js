const add = (a, b) => {
  console.log(a, b);
  return a + b;
};

describe("group 1", () => {
  test("test 1", () => {
    expect(add(1, 1)).toBe(2);
  });
});
