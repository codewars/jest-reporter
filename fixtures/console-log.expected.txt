
<DESCRIBE::>group 1

<IT::>test 1

<PASSED::>Test Passed

<COMPLETEDIN::>2

<COMPLETEDIN::>

<LOG::console.log at add (/fixtures/console-log.js:2:11)>1 1
